content/frameworks.yaml

```diff
@@ -75,12 +87,6 @@ frameworkGroups:
         description: >
           A high-performance feature editing framework for deck.gl.
 
-      - name: math.gl
```

Comment:
We should probably 
- replace nebula.gl with deck.gl-community
- add probe.gl?